go/internal/ble-driver/BleInterface_darwin.m

```diff
@@ -20,57 +20,54 @@
 os_log_t OS_LOG_BLE = nil;
 
 void handleException(NSException* exception) {
-    os_log_error(OS_LOG_BLE, "Unhandled exception %{public}@", exception);
+    os_log_error(OS_LOG_BLE, "🔴 Unhandled exception %{public}@", exception);
 }
 
 BleManager* getManager(void) {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        os_log(OS_LOG_BLE, "getManager() initialize");
-        manager = [[BleManager alloc] initScannerAndAdvertiser];
+        os_log(OS_LOG_BLE, "🟢 getManager() initialize");
+        manager = [[[BleManager alloc] initScannerAndAdvertiser] autorelease];
```

Comment:
If I remember correctly: `autorelease` is only useful if used in conjunction with an `autoreleasepool`. But I'm not sure.

Reply:
Here is the `autoreleasepool` :)
https://github.com/berty/berty/pull/3389/files/a253a9c3f05a92a056b0468e71c04ea0134ace1b#diff-c22f6876520e4ee313ba1188699ae63a388fcad1b2ef82f655015878946b62e2R38

Reply:
Haha oops I missed it! :)
Alright 👍